modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main/FDSPropsTransformer/ViewVersionHistoryFDSPropsTransformer.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {IInternalRenderer} from '@liferay/frontend-data-set-web';
+
+import createAssetAction from './actions/createAssetAction';
+import multipleFilesUploadAction from './actions/multipleFilesUploadAction';
+import AuthorRenderer from './cell_renderers/AuthorRenderer';
+import NameRenderer from './cell_renderers/NameRenderer';
+import TypeRenderer from './cell_renderers/TypeRenderer';
+import addOnClickToCreationMenuItems from './utils/addOnClickToCreationMenuItems';
+
+const ACTIONS = {
+	createAsset: createAssetAction,
+	uploadMultipleFiles: multipleFilesUploadAction,
+};
+
+export default function ViewVersionHistoryFDSPropsTransformer({
+	creationMenu,
+	itemsActions = [],
+	...otherProps
+}: {
+	creationMenu: any;
+	itemsActions?: any[];
+	otherProps: any;
+}) {
+	return {
+		...otherProps,
+		creationMenu: {
+			...creationMenu,
+			primaryItems: addOnClickToCreationMenuItems(
+				creationMenu.primaryItems,
+				ACTIONS
+			),
+		},
+		customRenderers: {
+			tableCell: [
+				{
+					component: AuthorRenderer,
+					name: 'authorTableCellRenderer',
+					type: 'internal',
+				} as IInternalRenderer,
+				{
+					component: NameRenderer,
+					name: 'nameTableCellRenderer',
+					type: 'internal',
+				} as IInternalRenderer,
+				{
+					component: TypeRenderer,
+					name: 'typeTableCellRenderer',
+					type: 'internal',
+				} as IInternalRenderer,
+			],
+		},
+		itemsActions: itemsActions.map((action) => {
```

Comment:
I think this is the only part that you need from the propsTransformer, the `itemActions`, because the only action that need to do something (you control when it is visible). The rest, `customRenderers` and `creationMenu` can be removed, isn't it?

Reply:
No at all, I see that I can delete creationMenu because I don't need a creaction menu here :(. But I forgot to add cell renderes in the other PR and I need them, but I see that I can delete typeTableCellRenderer also.

thanks for your time @ambrinchaudhary